src/wiki/plugins/linknetwork/templates/wiki/plugins/linknetwork/linknetwork.html

```diff
@@ -0,0 +1,70 @@
+{% extends "wiki/base.html" %}
+{% load wiki_tags i18n humanize %}
+
+{% block wiki_pagetitle %}{% trans "Link Network" %}{% endblock %}
+
+{% block wiki_contents %}
+
+<h1>{% trans "Link Network" %}</h1>
+
+{% spaceless %}
```

Comment:
Curious quick comment (can review more thoroughly soon) : Why the `{% spaceless %}`?

Reply:
Ehm. I think I copied another list template – maybe globalhistory? – and adapted it minimally, because I am still new to Django's template language. I assume this is an artefact of that process.

Reply:
Ah okay :) It's nice to keep the spaces normally - `{% spaceless %}` is mostly for things where spaces should be removed, often because you have them for readability in the template, but once rendered they're wrong (examples: text emails and things inside `<textarea>`)